<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1165</width>
    <height>801</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FePEST Prior Information Dashboard</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>791</width>
      <height>82</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0,0">
     <item row="0" column="0">
      <widget class="QLabel" name="labelInputFilePath_2">
       <property name="text">
        <string>SEN File</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEditInputFilePath">
       <property name="text">
        <string>D:/Repositories/deber1-stor/41801573-2_Freeport_DRC/work/model/Localmodels/Tenke/femdata/Tenke_2014_r136_PEST_001/tenke_2014_r136_pest.sen</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QToolButton" name="toolButtonSelectInputFile">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QToolButton" name="toolButtonSelectInputFile_2">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineEditInputFilePath_2">
       <property name="text">
        <string>D:/Repositories/deber1-stor/41801573-2_Freeport_DRC/work/model/Localmodels/Tenke/femdata/Tenke_2014_r136_PEST_001/tenke_2014_r136_pest.pst</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labelInputFilePath">
       <property name="text">
        <string>PST File</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="pushButtonRefresh">
       <property name="text">
        <string>Load</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="1" column="1">
        <widget class="QSpinBox" name="spinBox_IterationNumber">
         <property name="toolTip">
          <string>Choose the iteration to be loaded here</string>
         </property>
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::UpDownArrows</enum>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Iteration No. </string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="labelInputFilePath_3">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>JCO File</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="lineEditInputFilePath_3">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QToolButton" name="toolButtonSelectInputFile_3">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>1091</width>
      <height>581</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <widget class="QWidget" name="tab_slaves">
     <attribute name="title">
      <string>Preferred Values</string>
     </attribute>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QTreeWidget" name="treeWidgetSlaves">
        <property name="maximumSize">
         <size>
          <width>1061</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="indentation">
         <number>8</number>
        </property>
        <property name="sortingEnabled">
         <bool>true</bool>
        </property>
        <attribute name="headerDefaultSectionSize">
         <number>125</number>
        </attribute>
        <attribute name="headerMinimumSectionSize">
         <number>10</number>
        </attribute>
        <column>
         <property name="text">
          <string>Parameter</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Current Value</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Current Sensitivity</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Lower Bound</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Preferred</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Upper Bound</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="tab_statistics">
     <attribute name="title">
      <string>SEN File</string>
     </attribute>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QPlainTextEdit" name="plainTextEdit">
        <property name="font">
         <font>
          <family>Courier New</family>
         </font>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>PST File</string>
     </attribute>
     <layout class="QGridLayout" name="gridLayout_8">
      <item row="0" column="0">
       <widget class="QPlainTextEdit" name="plainTextEdit_PstFile">
        <property name="font">
         <font>
          <family>Courier New</family>
         </font>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>GENLINDPRED</string>
     </attribute>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="1">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="pushButtonRunGenlinpred">
        <property name="text">
         <string>Run GENLINPRED</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QTabWidget" name="tabWidget_2">
        <property name="currentIndex">
         <number>2</number>
        </property>
        <widget class="QWidget" name="tab_4">
         <attribute name="title">
          <string>Output</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_6">
          <item row="0" column="0">
           <widget class="QPlainTextEdit" name="plainTextEdit_Genlinpredprompt">
            <property name="font">
             <font>
              <family>Courier New</family>
             </font>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_5">
         <attribute name="title">
          <string>Files</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_7">
          <item row="0" column="0">
           <widget class="QPlainTextEdit" name="plainTextEdit_Genlinpredout">
            <property name="font">
             <font>
              <family>Courier New</family>
             </font>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_6">
         <attribute name="title">
          <string>PREDVAR1</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_9">
          <item row="0" column="0">
           <layout class="QGridLayout" name="gridLayout_PREDVAR"/>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Parameter Values</string>
     </attribute>
     <widget class="QWidget" name="widget" native="true">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>120</y>
        <width>211</width>
        <height>81</height>
       </rect>
      </property>
     </widget>
     <widget class="QTreeWidget" name="treeWidgetParameterState">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>1031</width>
        <height>501</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>1061</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::DefaultContextMenu</enum>
      </property>
      <property name="dragDropMode">
       <enum>QAbstractItemView::NoDragDrop</enum>
      </property>
      <property name="alternatingRowColors">
       <bool>false</bool>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::MultiSelection</enum>
      </property>
      <property name="indentation">
       <number>8</number>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <attribute name="headerDefaultSectionSize">
       <number>125</number>
      </attribute>
      <attribute name="headerMinimumSectionSize">
       <number>10</number>
      </attribute>
      <column>
       <property name="text">
        <string>Parameter</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Sensitivity</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Reg.Obj.</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Identifiab.</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Parameter Values and Uncertainty Margins</string>
       </property>
      </column>
     </widget>
     <widget class="QCheckBox" name="checkBox_BoundBrackets">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>91</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Bound Bracket</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_Deviation">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>10</y>
        <width>121</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Deviation from PI</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_PreCalParamUncert">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>10</y>
        <width>121</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Pre-Cal Uncertainty</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_PostCalParamUncert">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>10</y>
        <width>121</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Post-Cal Uncertainty</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLCDNumber" name="lcdNumberTotalRuns">
    <property name="geometry">
     <rect>
      <x>1090</x>
      <y>10</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(55, 255, 0);
background-color:rgb(100,100,100);
</string>
    </property>
    <property name="smallDecimalPoint">
     <bool>true</bool>
    </property>
    <property name="digitCount">
     <number>6</number>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Flat</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>974</x>
      <y>10</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Number of iterations:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_senFileNotReadable">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>40</y>
      <width>110</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(255, 0, 0)</string>
    </property>
    <property name="text">
     <string>Can't read from SEN file</string>
    </property>
   </widget>
   <zorder>gridLayoutWidget</zorder>
   <zorder>lcdNumberTotalRuns</zorder>
   <zorder>label_2</zorder>
   <zorder>tabWidget</zorder>
   <zorder>label_senFileNotReadable</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1165</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open ...</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
